cip.go

```diff
@@ -93,6 +93,10 @@ func main() {
 		"minimal-snapshot",
 		false,
 		"(only works with -snapshot) discard tagless images from -snapshot output if they are referenced by a manifest list")
+	flattenedSnapshotPtr := flag.Bool(
+		"flattened-snapshot",
+		false,
+		"(only works with -snapshot) instead of printing YAML, print all images by their full digest reference, sorted lexicographically")
```

Comment:
Maybe rename to `--output` or something and treat this like an output format?

Reply:
(You need only support it for the snapshot case, for now)

Reply:
Hmm, good point. Ack